docs/data/routes/docs/nextjs/getting-started/walkthrough-dotnetnew/en.md

```diff
@@ -3,4 +3,193 @@ name: walkthrough-dotnetnew
 routeTemplate: ./data/component-templates/article.yml
 title: Walkthrough with `dotnet new`
 ---
-# Walkthrough: with `dotnet new`
\ No newline at end of file
+# Walkthrough: Using the .NET starter template for JSS Next.js projects
+
+Sitecore provides a Getting Started template for Sitecore developers that want to try out  Sitecore JSS with [Sitecore Containers](https://containers.doc.sitecore.com/), the [Sitecore Next.js SDK](https://jss.sitecore.com/), and [Sitecore Content Serialization](https://doc.sitecore.com/developers/101/developer-tools/en/sitecore-content-serialization.html). 
+
+For simplicity, this solution does not implement Sitecore Helix conventions for solution architecture. As you begin building your Sitecore solution, you should review [Sitecore Helix](https://helix.sitecore.net/) and the [Sitecore Helix Examples](https://sitecore.github.io/Helix.Examples/) for guidance on implementing a modular solution architecture.
+
+The template includes: 
+
+* A `docker-compose` environment for a Sitecore XP0 topology with a Next.js rendering host.
+
+ > The included `docker-compose.yml` is a stock XP0 environment from the Sitecore Container Support Package. All changes/additions for this solution are included in the `docker-compose.override.yml`.
+
+* Scripted invocation of `jss create` and `jss deploy` to initialize a Next.js application.
+
+* Sitecore Content Serialization configuration.
+
+* An MSBuild project for deploying configuration and code into the Sitecore Content Management role. (see `src\platform`).
+
+## Support
+
+Sitecore supports the template output as provided. Once changed or amended, the solution becomes a custom implementation and is subject to limitations as defined in Sitecore's [scope of support](https://kb.sitecore.net/articles/463549#ScopeOfSupport).
+
+## Create a solution from the Getting Started template
+
+To create a solution from the Getting Started template, you must complete the following procedures:
+
+- [Install the prerequisites](#install-the-prerequisites)
+- [Install the template](#install-the-template)
+- [Create the MyProject solution](#create-the-myproject-solution)
+- [Rebuild search indexes](#rebuild-search-indexes)
+- [Add a language and import (Optional)](#add-a-language-and-import-optional)
+
+### Install the prerequisites
```

Comment:
Should the Sitecore license file be mentioned in pre-reqs?

Reply:
Yes. We also need powershell 5.1. I re-worked the pre-reqs section to include both.